### PR TITLE
proper casing for all packages

### PR DESCRIPTION
**Pull request by Teju-Manchenella**
New toLowerCase map across website, service, and crawler. Addresses issue #[237](https://github.com/clearlydefined/service/issues/157)

### CODE REVIEW

**Comment by Teju-Manchenella** [comment]
Issue open [#123](https://github.com/clearlydefined/crawler/issues/123)

**Comment by jeffmcaffer** [comment]
@dabutvin you might be interested in merging these changes into the work you are doing on case sensitivity in the service and crawler

File: src/api/clearlyDefined.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ function getServiceDefaultUrl() {
   switch (process.env.NODE_ENV) {
     case 'development':
       //TODO this need to be replaced when using a local enviroment, it should become API_LOCAL
-      return API_DEVELOP
+      return API_LOCAL
```
[the remark this answers is not in the document]
Comment (AlexWebYourmind) [reply src/api/clearlyDefined.js]: We are addressing  the way env variables should be set in https://github.com/clearlydefined/website/issues/257

**Comment by jeffmcaffer** [comment]
this is now obsoleted by a different set of changes.

**Pull request closed without merging**